docs/src/main/sphinx/admin/properties-client-protocol.md

```diff
@@ -0,0 +1,183 @@
+# Client protocol properties
+
+The following sections provide a reference for all properties related to the
+[client protocol](/client/client-protocol). 
+
+(prop-protocol-spooling)=
+## Spooling protocol
+
+The following properties are related to the [](protocol-spooling).
+
+### `protocol.spooling.enabled`
+
+- **Type:** [](prop-type-boolean)
+- **Default value:** `true` (is that automatically configured if shares secret or so is set up)
```

Comment:
no, you need to explicitly enable that will allow you to configure other things (separate config that is bound when enabled)